index.html

```diff
@@ -7,8 +7,25 @@
   <link href='css/mapbox-gl.css' rel='stylesheet' />
   <link rel='stylesheet' type='text/css' href='css/style.css'>
   <link rel='shortcut icon' href='img/nypl.png'>
+  <style>
+    .archive-banner {
```

Comment:
Can you add a rule to make the links colored black or blue? It's not readable right now:
![Screenshot 2024-11-14 at 1 32 24 PM](https://github.com/user-attachments/assets/1db838e0-3d71-49bf-958f-1d4997295fb0)